RELEASE-NOTES.txt

```diff
@@ -1,8 +1,12 @@
 Unreleased
 ---
+
+1.67.0
+---
 * [**] Adds Clipboard Link Suggestion to Image block and Button block [https://github.com/WordPress/gutenberg/pull/35972]
 * [*] [Embed block] Included Link in Block Settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4189]
 * [**] Fix tab titles translation of inserter menu [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4248]
+* [*] [Media & Text block] Fix an issue where the text font size would be bigger than expected in some cases [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4252]
 * [**] [Gallery block] When a gallery block is added, the media options are auto opened for v2 of the Gallery block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4277]
 * [**] Fix undo/redo functionality in links when applying text format [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4290]
```

Comment:
[The PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4290) related to this entry is pointing to `1.68.0` milestone, as it was merged right after the release was cut, so it should be part of the `Unreleased` section.

Reply:
Thanks for spotting that @fluiddot! It looks like GitHub automatically added it to the wrong section after I recently merged develop into this branch. I corrected this in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4296/commits/7d105694612a5b1b8d689a1bdd6d91d9e17f2294.